tests/test_taking_a_vacation.py

```diff
@@ -57,18 +56,60 @@ def test_plane_ride_cost_negative():
     (8, 40*8-50),
     (365, 40*365-50)
 ])
-def test_rental_car_cost(input, expected):
-    assert rental_car_cost(input) == expected
+def test_rental_car_cost(input_days, expected):
+    assert rental_car_cost(input_days) == expected
 
 
-@pytest.mark.parametrize("input", [-1, 0, "blah", False, None])
-def test_rental_car_cost_negative(input):
+@pytest.mark.parametrize("input_nights", [-1, 0, "blah", False, None])
+def test_rental_car_cost_negative(input_nights):
     try:
-        hotel_cost(input)
+        hotel_cost(input_nights)
         assert False, "Expected an exception from rental_car_cost"
     except AssertionError as e:
-        assert str(e) == str(input) + " was not int or positive integer > 0"
+        assert str(e) == str(input_nights) + " was not int or positive integer > 0"
+
+
+@pytest.mark.parametrize("input_city, input_days, input_spending_money, expected", [
+    ("Charlotte", 1, 100, 463),
+    ("Tampa", 7, 100, 1530),
+    ("Pittsburgh", 31, 100, 5852),
+    ("Los Angeles", 365, 100, 66225)
+])
+def test_trip_cost(input_city, input_days, input_spending_money, expected):
+    assert trip_cost(input_city, input_days, input_spending_money) == expected
 
 
+@pytest.mark.parametrize("input_spending_money", [-1, "blah", None, False])
+@pytest.mark.parametrize("input_days", [-1, "blah", False, None])
+@pytest.mark.parametrize("input_city", ["Sochi"])
```

Comment:
don't use parametrize when you have only 1 test value, just put that value in the call to the function under test.